website/templates/project/statistics.mako

```diff
@@ -5,6 +5,16 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag" class="ad_block_display">
+  <div class="m-b-md p-md osf-box-lt box-round text-center">The use of adblocking software may prevent site analytics from loading properly.</div>
+</div>
+<script src="/static/js/ads.js"></script>
+<script>
+if( window.canRunAds === undefined ){
+  $('.ad_block_display').css({'display':'block'});
```

Comment:
This one was my suggestion for reusability. If we want to do the same sort of thing with adblocking, there's no reason why we wouldn't do it to everything on that page that we want shown.